packages/manager/modules/octavia-load-balancer/src/create/constants.js

```diff
@@ -0,0 +1,33 @@
+export const PRODUCT_LINK = {
+  FR: 'https://www.ovhcloud.com/fr/public-cloud/load-balancer/',
+  DE: 'https://www.ovhcloud.com/de/public-cloud/load-balancer/',
+  ES: 'https://www.ovhcloud.com/es-es/public-cloud/load-balancer/',
+  WE: 'https://www.ovhcloud.com/en-ie/public-cloud/load-balancer/',
+  IE: 'https://www.ovhcloud.com/en-ie/public-cloud/load-balancer/',
+  IT: 'https://www.ovhcloud.com/it/public-cloud/load-balancer/',
+  NL: 'https://www.ovhcloud.com/nl/public-cloud/load-balancer/',
+  PL: 'https://www.ovhcloud.com/pl/public-cloud/load-balancer/',
+  PT: 'https://www.ovhcloud.com/pt/public-cloud/load-balancer/',
+  GB: 'https://www.ovhcloud.com/en-gb/public-cloud/load-balancer/',
+  CA: 'https://www.ovhcloud.com/en-ca/public-cloud/load-balancer/',
+  QC: 'https://www.ovhcloud.com/fr-ca/public-cloud/load-balancer/',
+  WS: 'https://www.ovhcloud.com/es/public-cloud/load-balancer/',
+  MA: 'https://www.ovhcloud.com/fr-ma/public-cloud/load-balancer/',
+  SN: 'https://www.ovhcloud.com/fr-sn/public-cloud/load-balancer/',
+  TN: 'https://www.ovhcloud.com/fr-tn/public-cloud/load-balancer/',
+  AU: 'https://www.ovhcloud.com/en-au/public-cloud/load-balancer/',
+  SG: 'https://www.ovhcloud.com/en-sg/public-cloud/load-balancer/',
+  ASIA: 'https://www.ovhcloud.com/asia/public-cloud/load-balancer/',
+  US: 'https://www.ovhcloud.com/en/public-cloud/load-balancer/',
+};
+
+export const TRACKING_CHAPTER_1 = 'PublicCloud';
```

Comment:
**nitpick:** This should be set only once per app and not repeat in all screens.
Refer "Public Cloud" app.

Reply:
It is already done in octavia app : https://github.com/ovh/manager/blob/ac35d4ca2e4660a341a5c254ba5e706aeab814fc/packages/manager/apps/octavia-load-balancer/src/app.module.js#L29

But this only work for page display tracking event not for the click event. Maybe I am missing something